modules/samtools/merge/main.nf

```diff
@@ -22,14 +22,14 @@ process SAMTOOLS_MERGE {
     tuple val(meta), path(bams)
 
     output:
-    tuple val(meta), path("*merged.bam"), emit: merged_bam
-    path  "*.version.txt"              , emit: version
+    tuple val(meta), path("${prefix}.bam"), emit: merged_bam
```

Comment:
```suggestion
    tuple val(meta), path("${prefix}.bam"), emit: bam
```

Reply:
May need to change the `output` options in `meta.yml` too

Reply:
I think it should be fine that way